Client/Cards/create/modulr.spec.ts

```diff
@@ -0,0 +1,43 @@
+import * as dotenv from "dotenv"
+import * as pax2pay from "../../../index"
+import { factory } from "./factory"
+
+dotenv.config()
+jest.setTimeout(200000)
+
+describe("pax2pay.cards.create modulr", () => {
+	const client = pax2pay.Client.create(process.env.url)
+	beforeAll(
+		async () =>
+			await client?.auth.login({
+				username: process.env.username ?? "user",
+				password: process.env.password ?? "password",
+			})
+	)
+	for (const currency of ["EUR", "GBP"])
```

Comment:
modulr also has pln and usd

Reply:
can't find those accounts from searching